LDMP/main_widget.py

```diff
@@ -36,6 +39,36 @@ def __init__(self, parent=None):
         super(MainWidget, self).__init__(parent)
 
         self.setupUi(self)
+        self.setupAlgorithsTree()
+
+    def setupAlgorithsTree(self):
+        # setup algoriths and it's hierarchy
+        tree = AlgorithmGroup(name='root', name_details='root detauils', group=None, algorithms=[])
```

Comment:
Please take care of these typos

```suggestion
    def setupAlgorithmsTree(self):
        # setup algorithms and their hierarchy
        tree = AlgorithmGroup(name='root', name_details='root details', group=None, algorithms=[])
```